src/Datadog.Trace/Tracer.cs

```diff
@@ -119,7 +119,8 @@ internal Tracer(TracerSettings settings, ITraceWriter traceWriter, ISampler samp
             _scopeManager = scopeManager ?? new AsyncLocalScopeManager();
             Sampler = sampler ?? new RuleBasedSampler(new RateLimiter(Settings.MaxTracesSubmittedPerSecond));
 
-            _propagator = ContextPropagatorBuilder.BuildPropagator(Settings.Propagator, TraceIdConvention);
+            var propagators = ContextPropagatorBuilder.BuildPropagators(Settings.Propagators, TraceIdConvention);
```

Comment:
Please note the upcoming configuration issue here. PropagatorType should be matched with TraceIdConvention (eg: we can go with name:value based configuration key), cause single convention type could not be accepted for all of the propagators. (See PR #102 [comment](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/102#pullrequestreview-626175666))

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/102#discussion_r607669452

Reply:
Current result: no support for different TraceIds in same time